snuba/datasets/ddl.py

```diff
@@ -0,0 +1,9 @@
+class DDL(object):
+    def __init__(self, schemas=[]):
+        self.__schemas = schemas
+
+    def create_statements(self):
```

Comment:
We started using type hints in Snuba. Please add them to all new code
Also I would rename it to "get_create_statements" otherwise one would believe this method would create some statements